### DISCUSSION TITLE
Remove file without removing it from file system?

### DISCUSSION THREAD

**Discussion by flextra19** [community/community#30104] [Other Feature Feedback, Questions, & Ideas] [answered]
Please help me with one problem.
I gonna remove file from git without removing it from my file system.
I know it's possible, but don't know the way.
Please guide me.  Thank you.

**Comment by bojanterzic529** [comment, marked as answer]
One has to be careful during a git add, else you may end up adding files that you didn’t want to commit. However, git rm will remove it from both your staging area (index), as well as your file system (working tree), which may not be what you want.

Instead, use git reset:

git reset filename          # or

echo filename >> .gitingore # add it to .gitignore to avoid re-adding it

This means that git reset <paths> is exactly the opposite of git add <paths>.

**Reply by flextra19** [reply]
Great help!
